gooddata-java/src/test/java/com/gooddata/sdk/service/project/ProjectServiceAT.java

```diff
@@ -144,6 +146,16 @@ public void addUsersToProject() {
         assertThat(user2, is(notNullValue()));
     }
 
+    @Test(groups = {"project", "isolated_domain"}, dependsOnMethods = "addUsersToProject")
+    public void listProjectsForUser() {
+        final ProjectService projectService = gd.getProjectService();
+
+        final PageBrowser<Project> projects = projectService.listProjects(account1, new CustomPageRequest());
```

Comment:
It would be great to test both new methods - this test case you wrote is great for the second method without start page. Test of method with start page would be great to actually verify this feature ;)

Reply:
so removed customPage, one tests will test both now.